docker-compose.yml

```diff
@@ -13,7 +13,7 @@ services:
     image: node:14-alpine
```

Comment:
heads up: 

Not everything in here is unit tested. In the interests of speed, this PR is going in without some unit tests. These will follow in my next PR.